.github/workflows/my-ci-pipeline.yml

```diff
@@ -19,4 +17,34 @@ jobs:
     - run: ls
     - run: touch .env # creating empty env file coz docker compose needs one
     - run: docker build --target test -t todo-app:test . 
-    - run: docker run --env-file ./.env.test todo-app:test
\ No newline at end of file
+    - run: docker run --env-file ./.env.test todo-app:test
+
+  dockerhub:
+    name: Build and image
+    runs-on: ubuntu-latest
+    needs: build
+    if: github.ref_name == 'module8_updated'
+    steps:
+    - run: echo "Publishing!"
+    - uses: actions/checkout@v2
+    - name: docker build
+      run: docker build -t zorion75/todo-app-module8 .
+    - name: login
+      run: echo ${{secrets.DOCKER_HUB_TOKEN}} | docker login --username zorion75 --password-stdin
+    - name: push image to docker hub
+      run: docker push zorion75/todo-app-module8
+
+  heroku:
+    name: deplay to heroku
+    runs-on: ubuntu-latest
+    needs: dockerhub
+    steps:
+    - uses: actions/checkout@v2
+    - uses: akhileshns/heroku-deploy@v3.12.12
+      with:
+        heroku_api_key: ${{secrets.HEROKU_API}}
+        heroku_app_name: "todo-app-kat-module8"
```

Comment:
I can work it out from this name, but it would be good to add a link in your Readme to the deployed site <http://todo-app-kat-module8.herokuapp.com/>